specification/devcenter/DevCenter/DevCenter/routes.tsp

```diff
@@ -14,23 +14,4 @@ interface DevCenter {
 
   @doc("Gets a project.")
   getProject is StandardResourceOperations.ResourceRead<Project>;
-
-  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Does not fit any standard operation pattern since DevBox has a different path"
-  @doc("Lists Dev Boxes that the caller has access to in the DevCenter.")
-  @route("/devboxes")
-  @get
-  listAllDevBoxes is Azure.Core.Foundations.Operation<{}, DevBoxListResult>;
-
-  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Does not fit any standard operation pattern since DevBox has a different path"
-  @doc("Lists Dev Boxes in the Dev Center for a particular user.")
-  @route("/users/{userId}/devboxes")
-  @get
-  listAllDevBoxesByUser is Azure.Core.Foundations.Operation<
```

Comment:
Why did we move these operations out of the DevCenter interface? This is kind of breaking for docs at least